packages/asset-swapper/src/utils/market_operation_utils/orders.ts

```diff
@@ -199,6 +199,9 @@ function getBridgeAddressFromFill(fill: CollapsedFill, opts: CreateOrderFromPath
             return opts.contractAddresses.shellBridge;
         case ERC20BridgeSource.Dodo:
             return opts.contractAddresses.dodoBridge;
+        case ERC20BridgeSource.CryptoCom:
+            // HACK: deploy using unique address
+            return opts.contractAddresses.sushiswapBridge;
```

Comment:
Is the only side effect of this that it would show up as sushi volume in analytics?

Reply:
Yeah and avoid paying the overhead of FQT<-->Bridge transfer costs